Probesys/bileto
# fix: Track changes to the tickets' ongoing contracts

## Related issue(s)



https://github.com/Probesys/bileto/issues/460

## How to test manually



- create a ticket
- create two contracts in the same organization
- add a contract to the ticket
- change the contract of the ticket
- remove the contract of the ticket
- check that all three events appear in the ticket's timeline

## Checklist
